pkgs/development/ocaml-modules/nice_parser/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  menhir,
+  ppx_jane,
+  base,
+  stdio,
+  odoc,
+}:
+
+buildDunePackage rec {
+  pname = "nice_parser";
+  version = "1.0.0";
+  minimalOCamlVersion = "4.05";
```

```suggestion
  minimalOCamlVersion = "4.07";
```

In `nixpkgs`, the `stdio` library is not available for OCaml < 4.07.

---

Got it!